mods/cnc/rules/civilian-desert.yaml

```diff
@@ -4,8 +4,8 @@ V20:
 		UseTargetableCellsOffsets: false
 		TargetableOffsets: -840,-512,0, 0,0,0, -840,512,0
 		Type: Rectangle
-			TopLeft: -1024, -512
-			BottomRight: 1024, 896
+			TopLeft: 1024,-512
+			BottomRight: -1024,896
```

Comment:
This puts the point to the left of the `TopLeft`.  The rule really should switch `TopLeft` and `BottomRight` (which could be a straight rename if `Y` indeed also does need to be inverted).